collections/crowdsecurity/endlessh.yaml

```diff
@@ -0,0 +1,13 @@
+#the list of parsers it contains
+parsers:
+  - ./parsers/crowdsecurity/endlessh-logs.yaml
+  - crowdsecurity/dateparse-enrich
+scenarios:
+  - ./scenarios/crowdsecurity/endlessh-bf.yaml
+description: "endlessh support : logs parser and brute-force detection"
+author: bamx23
```

Comment:
Unfortunately the name of the author must match the collection path for now (it is parsed by the hub app to generate the commands)
 - either you put `crowdsecurity` in the author name
 - or you keep `bamx23` but you add a `bamx23` directory for your parsers, scenarios and collection

Reply:
`crowdsecurity` is fine, fixed